linkis-dist/package/conf/linkis.properties

```diff
@@ -89,4 +89,12 @@ linkis.session.redis.password=test123
 # redis sso switch
 linkis.session.redis.cache.enabled=false
 wds.linkis.workspace.filesystem.owner.check=true
-wds.linkis.workspace.filesystem.path.check=true
\ No newline at end of file
+wds.linkis.workspace.filesystem.path.check=true
+
+#linkis token
+linkis.configuration.linkisclient.auth.token.value=BML-AUTH
+wds.linkis.client.common.tokenValue=BML-AUTH
+wds.linkis.bml.auth.token.value=BML-AUTH
+wds.linkis.context.client.auth.value=BML-AUTH
+wds.linkis.errorcode.auth.token=BML-AUTH
```

Comment:
wds.linkis.filesystem.token.value need be add

Reply:
wds.linkis.server.dsm.auth.token.value

Reply:
modified